canonical-web-and-design/jp.ubuntu.com
# Update marketo endpoint

## Done

- Post forms to https://ubuntu.com/marketo/submit
- Decapitalise input names
- Update hidden inputs
- Remove country input

## QA

1. Go to https://jp-ubuntu-com-370.demos.haus/contact-us?openstack
2. Fill in the form and submit it
3. Check it's submitted and redirect to a thank you page on jp.ubuntu.com
4. Check the submitted data is on the list in marketo (form ID is 2549)

## Issue / Card

Fixes [#4735](https://github.com/canonical-web-and-design/web-squad/issues/4735)
